azext_iot/central/commands_device.py

```diff
@@ -262,6 +302,287 @@ def get_command_history(
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.v1.value,
+):
+    from uuid import uuid4
+
+    if api_version != ApiVersion.v1_1_preview.value:
+        raise InvalidArgumentValueError(
+            (
+                "Adding children devices to IoT Edge is still in preview "
+                "and only available for Api version >= 1.1-preview. "
+                'Please pass the right "api_version" to the command.'
+            )
+        )
+
+    provider = CentralDeviceProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+    rel_name = get_downstream_rel_name(
+        cmd,
+        app_id=app_id,
+        device_id=device_id,
+        token=token,
+        central_dns_suffix=central_dns_suffix,
+        api_version=api_version,
+    )
+
+    if not rel_name:
+        raise ResourceNotFoundError(
+            f'Relationship name cannot be found in the template for device with id "{device_id}"'
+        )
+
+    return [
+        provider.add_relationship(
+            device_id=device_id,
+            target_id=child_id,
+            rel_id=str(uuid4()),
+            rel_name=rel_name,
+        )
+        for child_id in children_ids
+    ]
+
+
+def remove_children(
+    cmd,
+    app_id: str,
+    device_id: str,
+    children_ids: List[str],
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.v1.value,
+):
+
+    provider = CentralDeviceProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+    rel_name = get_downstream_rel_name(
+        cmd,
+        app_id=app_id,
+        device_id=device_id,
+        token=token,
+        central_dns_suffix=central_dns_suffix,
+        api_version=api_version,
+    )
+
+    if not rel_name:
+        raise ResourceNotFoundError(
+            f'Relationship name cannot be found in the template for device with id "{device_id}"'
+        )
+
+    rels = provider.list_relationships(
+        device_id=device_id, rel_name=rel_name, central_dns_suffix=central_dns_suffix
+    )
+    deleted = []
+    for rel in rels:
+        if rel.target in children_ids:
+            deleted.append(
+                provider.delete_relationship(
+                    device_id=device_id,
+                    rel_id=rel.id,
+                    central_dns_suffix=central_dns_suffix,
+                )
+            )
+
+    if not deleted:
+        raise ForbiddenError(f"Childs {children_ids} cannot be removed.")
+
+    return deleted
+
+
+def get_edge_device(
+    cmd,
+    app_id: str,
+    device_id: str,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.v1.value,
+) -> Any:
+    provider = CentralDeviceProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+
+    def raise_error():
+        raise InvalidArgumentValueError(
+            "The specified device Id does not identify an IoT Edge device."
```

Comment:
nit (adding 'as'): "The specified device Id does not identify **as** an IoT Edge device."